package.json

```diff
@@ -208,7 +208,8 @@
   "overrides": {
     "@babel/traverse": "7.23.2",
     "@nx/devkit": "17.0.2",
-    "debug": "4.3.4"
+    "debug": "4.3.4",
+    "rollup": "4.1.4"
```

Comment:
We're overriding the same version specified in the dependencies. Is this intentional?

Reply:
yes one of the rollup plugins has not caught up yet with v4 in its dependencies (but has no compatiblity issues)